HomeUI/src/views/apps/RegisterFluxApp.vue

```diff
@@ -4107,10 +4107,11 @@ export default {
      */
     async encryptMessage(message, encryptionKeys) {
       try {
+        const openpgp = await getOpenPGP();
         const publicKeys = await Promise.all(encryptionKeys.map((armoredKey) => openpgp.readKey({ armoredKey })));
         console.log(encryptionKeys);
         console.log(message);
-        const pgpMessage = await openpgp.createMessage({ text: message.replace('\\“', '\\"') });
+        const pgpMessage = await openpgp.createMessage({ text: message.replace('\\"', '\\"') });
```

Comment:
replace('\\“', '\\"') has been changed to replace('\\"', '\\"') that has no effect.